### PR TITLE
Add history incidence maps

### LINKED ISSUE #56: Map with specific date
I would also love to have a parameter for the maps, to see the map of a specific date.

_Originally posted by @CodeBrauer in https://github.com/marlon360/rki-covid-api/issues/51#issuecomment-753542673_

### PR DESCRIPTION
**Pull request by Rubber1Duck**
add the possibility to request all maps with parameter :date witch can be a date in form of 2021-11-01 or number of days to go in the past!
following requests are possible: (:date can be a date or days to go in the past)
 
- /map/districts/history/:date
- /map/distritcs-legend/history/:date
- /map/states/history/:date
- /map/states-legend/history/:date
- /map/states/hospitalisation/history/:date
- /map/states-legend/hospitalization/history/:date

Fix#56

### CODE REVIEW

File: src/responses/map.ts [hunk 1]
```diff
@@ -153,6 +158,124 @@ export async function StatesLegendMapResponse() {
     .toBuffer();
 }
 
+// Begin history map respones
+export async function DistrictsHistoryMapResponse(
+  mapType: string, // "map" for map without legend, "legendMap" for map with legend
```
Comment (marlon360) [inline src/responses/map.ts]: Let's use an `enum` for `mapType` and not a `string`

Comment (marlon360) [reply src/responses/map.ts]: Let's also add a default value for `mapType`. This ensures the response always returns something.

**Comment by Rubber1Duck** [comment]
enum .......... da musste ich mich erst belesen :-) ich hoffe du hast dir das so vorgestellt

**Comment by Rubber1Duck** [comment]
Anything else to do?

**Comment by Rubber1Duck** [comment]
@marlon360 everything is now adapted to the new circumstances! would you please take a look at it and/or approve it?!

**Review by marlon360 — APPROVED** [review]

**Pull request merged**